### ISSUE TITLE
Question about Valid result

### ISSUE THREAD

**Issue by NgoDoanKien** [NZWANG/MM-GTUNets#3]
Thanks for introducing such an impressive model.
I have tried to run your Model on training mode with 5 folds on my custom dataset. I has encountered a “strange” circumstance that in the 1st fold, accuracy and f1 score are always much lower than later folds. 
Details: 
1st Fold f1 ~70
2nd - 5th fold: f1 ~96-99
I have changed lr, epoch, node_dim but it still happened
Is that an evidence of data leakage? Or is it a normal phenomenon? 
Furthermore, I tested the model on a hold-out test set  (ensemble of 5 folds) and the f1 score is just equal to the 1st fold f1 score.

**Comment by clh0x123** [comment]
If you choose to use 5-fold cross-validation for your experiments, please adjust the corresponding validation set ratio accordingly. In 10-fold cross-validation, the ratio of the training set, validation set, and test set is approximately 8:1:1. If you have any questions about dataset splitting, please refer to lines 175 to 185 in utils/mydataloader.py.

**Comment by NgoDoanKien** [comment]
for data_split function, I set val_ratio=0. That is I only use train and val set with 5 folds split. I have commented all the test part. My dataset has about 1200 examples so IMO it's not due to the splitting stage. I also tried different random stage number too and the 1st fold alwsay has lower score than others

**Comment by clh0x123** [comment]
Based on your feedback, I believe that the possibility of data leakage can be essentially ruled out. Personally, I suggest that under the same experimental conditions, you might consider observing the performance of other models (such as  Ridge Classifier and SVM) on your dataset. Additionally, when making modifications to the original training framework, please carefully review all related sections to avoid introducing bugs.
